src/pages/iou/request/step/IOURequestStepDescription.tsx

```diff
@@ -64,8 +65,10 @@ function IOURequestStepDescription({
     const {inputCallbackRef} = useAutoFocusInput(true);
     // In the split flow, when editing we use SPLIT_TRANSACTION_DRAFT to save draft value
     const isEditingSplitBill = iouType === CONST.IOU.TYPE.SPLIT && action === CONST.IOU.ACTION.EDIT;
-    const currentDescription = isEditingSplitBill && !lodashIsEmpty(splitDraftTransaction) ? splitDraftTransaction?.comment?.comment ?? '' : transaction?.comment?.comment ?? '';
-    const descriptionRef = useRef(currentDescription);
+    const currentDescriptionInMarkdown = Parser.htmlToMarkdown(
+        isEditingSplitBill && !lodashIsEmpty(splitDraftTransaction) ? splitDraftTransaction?.comment?.comment ?? '' : transaction?.comment?.comment ?? '',
+    );
+    const descriptionRef = useRef(currentDescriptionInMarkdown);
```

Comment:
The case of transaction draft was not handled in the initial implementation.

Issue: https://github.com/Expensify/App/issues/59986
Title: Expense-In confirmation page, description saved as <> is not shown on reopening